stretch-assignment/index.js

```diff
@@ -1,24 +1,48 @@
 // When a block is clicked, it should go to the top of the stack.
 
-// document.getElementsByTagName('body').style.backgroundColor = 'black';
+const selectParent = document.querySelector('.blocks');
```

Comment:
Hey @KingAtoki it's not quite working the way its supposed to. I tried using .pop() / .unshift()   &   .prepend.  When you get a chance, could you take a look? Thanks.